modules/search-service-impl/src/main/java/org/opencastproject/search/impl/SearchServiceImpl.java

```diff
@@ -484,6 +500,46 @@ public boolean deleteSynchronously(final String mediaPackageId) throws SearchExc
     }
   }
 
+  /**
+   * Immediately removes the given series from the search service.
+   *
+   * @param seriesId
+   *          the series
+   * @return <code>true</code> if the series was deleted
+   * @throws SearchException
+   *           if deletion failed
+   * @throws UnauthorizedException
+   *           if the user did not have access to the series
+   * @throws NotFoundException
+   *           if the series did not exist
+   */
+  public boolean deleteSeriesSynchronously(String seriesId) throws SearchException, UnauthorizedException,
```

Comment:
Some of those exceptions are not actually thrown by this code, e.g. Unauthorized & NotFound